src/stylesheets/elements/_inputs.scss

```diff
@@ -82,7 +82,9 @@ textarea {
   height: 16rem;
 }
 
+// padding right added to avoid text overlapping with bg image
 select {
+  padding-right: 3.2rem;
```

Comment:
sure thing @msecret -- thanks for the link; i wasn't sure what convention we followed